src/compute-types/src/dyncfgs.rs

```diff
@@ -132,6 +132,14 @@ pub const PERSIST_SINK_OBEY_READ_ONLY: Config<bool> = Config::new(
     "Whether the compute persist_sink obeys read-only mode.",
 );
 
+/// Sets the max lifetime for replicas configured as an offset to the replica start time.
+/// Diffs generated at timestamps beyond the expiration time are dropped.
+pub const COMPUTE_REPLICA_EXPIRATION: Config<Duration> = Config::new(
+    "compute_replica_expiration",
+    Duration::ZERO,
```

Comment:
Should we set this to a reasonable default such as 21 days?

Reply:
Yes, we can, but we need to define the default value in LD before merging this PR.

Reply:
Yes. Generally, it's good to keep config defaults that are in the code somewhat close to what most prod envs will be getting from LD, because the default in the code determines what most of CI is testing.

Reply:
Done.